### ISSUE TITLE
web/desktop issue involving autohide upload button on element-call rooms chat

### ISSUE THREAD

**Issue by autoteelar** [vector-im/element-web#23604]
### Steps to reproduce

![image](https://user-images.githubusercontent.com/76745649/198010440-497d8666-73df-44df-94a5-ed53ced5ff44.png)
![image](https://user-images.githubusercontent.com/76745649/198010563-f8b38897-fbff-4687-9f03-1afd785b47d6.png)

as you can see here, the upload button dissapears after a certian width is attained, understandeable to a degree but i genuinely dont think it is necessary especially since i dont even have to make it that small for it to dissapear, id say make it activate when you make it even smaller, or hide the emoji button instead, im videocalling my face is the emoji, im more likely to press the upload button than the emoji button. i feel others would feel the same. 

tldr, upload button autohiding is pretty inconvinient.

#### What did you expect?
the upload button to be able to be clicked no matter what size during video calls
#### What happened instead?

it auto hid not even that far in, yet the emoji button which i feel is less important doesnt autohide, but honestly i would prefer this button to stay too, i dont really like autohiding buttons.

if there is a majority who does like autohiding buttons just make a toggle or something.

### Will you send logs?

No
